### DISCUSSION TITLE
Улучшения туториала забавные кейсы

### DISCUSSION THREAD

**Discussion by Vivioza5** [open-tutorials/cypress#46] [Ideas]
При прохождении [Туториал: забавные кейсы](https://www.epic1h.com/fun_cases_in_cypress) заметил в начале туториала небольшое нарушение логики при создании и запуске проекта.
1. Перед началом "Ты установил Tiny Web Server."  и дальше отсылается на первый туториал, но там такое не прописывали, может стоит проще указать команду по установке Tiny Web Server?
2.  Ты загрузил тестовое приложение в папку ~/apps и команда по загрузке этого приложения 

`npx wget -d apps/ https://raw.githubusercontent.com/open-tutorials/cypress/main/apps/fun-cases.html
`
 
Дальше по учебнику идет команда:

`npm i --save-dev node-wget qr-util`

Считаю логика нарушена, потому как без установки **node-wget** нельзя скачивать приложение в папку  ~/apps, а если юзер скачал уже, то значит этот модуль **node-wget**  уже установлен и прописывать в  команде установки пакетов нету смысла.
Для исправления такой логики может надо эти две строчки перенести и поставить после  команды установки пакетов например:

**1. Проверка QR кода**

- [x]  Установи нужные пакеты:

`npm i --save-dev node-wget qr-util`

Загрузи тестовое приложение в папку ~/apps

`npx wget -d apps/ https://raw.githubusercontent.com/open-tutorials/cypress/main/apps/fun-cases.html`

или просто убрать из 

`npm i --save-dev node-wget qr-util`

установку **node-wget**

**Comment by breslavsky** [comment]
Спасибо за ваши замечания. Пожалуйста, проверьте.

# 🙋‍ Перед началом

* Ты инициализировал чистый **Node.js** проект `%/projects/cypress/fun_cases`
* Ты установил Cypress `npm i cypress@9 --save-dev`
* Ты установил Tiny Web Server и Wget `npm i tiny-server node-wget --save-dev`
* Ты загрузил тестовое приложение в папку `~/apps`

```bash
npx wget -d apps/ https://raw.githubusercontent.com/open-tutorials/cypress/main/apps/fun-cases.html
```

Если забыл, глянь в наш [второй туториал](https://www.epic1h.com/best_selectors#1-подготовка-веб-сервера)

**Reply by Vivioza5** [reply]
Да это то, что я и предлагал

**Comment by breslavsky** [comment]
В следующий раз, пожалуйста, добавляйте ошибки и баги в задачи. Спасибо за поддержку!

**Reply by Vivioza5** [reply]
я спрашивал в общем чате в телеграме посоветовали в дискуссиях писать) и не хотел засорять комуникацию команды которая в задачах происходит
